diagnostic.md

```diff
@@ -7,36 +7,39 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:alexkarasik/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
+You go into that directory that you just cloned.
 
-<!-- Remove this comment and place your answer here. -->
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+You give the command git branch <branchname>.
+Then git checkout <branchname>.
+We can combine these steps with git checkout -b <branchname>.
+
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git diff will show us a summary of the changes we make.
```

Comment:
"git status" is actually what we're looking for since we want a (short) summary of the changes, whereas git diff will display every change in every file.